vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/ChoiceBlock.java

```diff
@@ -0,0 +1,34 @@
+package edu.asu.diging.vspace.core.model.impl;
+
+import javax.persistence.Entity;
+import javax.persistence.OneToOne;
+
+import edu.asu.diging.vspace.core.model.IChoice;
+import edu.asu.diging.vspace.core.model.IChoiceBlock;
+
+@Entity
+public class ChoiceBlock extends ContentBlock implements IChoiceBlock {
+
+    @OneToOne(targetEntity = Choice.class)
+    private IChoice choice;
```

Comment:
what about the other choices? for now a choice block should simply show all choices of a branching point.

Reply:
No, we are displaying all the choices on the top and giving an option "Add Choice" to be able to add the choices(as each individual choice blocks) where ever we want them to be added. Isn't it?

Reply:
hm, we definitely want an option to add all at once (since in most cases users probably want to add all in a block anyways and layouting that will be easier), but even if we won't do that now, keep that in mind. It probably makes most sense to let choiceblocks have a list of choices so they can be reused for that scenario, even if they right now only contain one choice. makes sense?

Reply:
Okay got it.